src/Components/GameScreenComponents/Board.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import Column from "./Column"
+import "../../styles/GameScreen.css"
+import {numOfRows} from "../../Constants";
+
+function Board(){
+    const rowNumbers = []
```

Comment:
I'd recommend using the [`Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) function here:
```ts
const rowNumbers = Array.from({length: numOfRows}, (_, i) => <div key={i}>{i+1}</div>)
```
Note the `key` prop on the `div`: for an explanation see [React docs](https://reactjs.org/docs/lists-and-keys.html)

Generally speaking, it's really helpful and common to use array methods like `map` and `filter` in React